docusaurus/docs/iOS/guides/push-notifications.md

```diff
@@ -1,3 +1,104 @@
 ---
 title: Push Notifications
----
\ No newline at end of file
+---
+
+# Push Notifications
+
+First step on a way to setting up push notifications is authentication.
+Stream supports both **Certificate-based provider connection trust (.p12 certificate)** and **Token-based provider connection trust (JWT)**. Token based authentication is the preferred way to setup push notifications. This method is easy to setup and provides strong security. You can find more on setting up authentication [here](https://getstream.io/chat/docs/php/push_ios/?language=swift).
+
+## Testing Push Notifications setup
+
+Once authentication is taken care of, you can test your setup as explained below.
+
+First of all make sure you have at least one [device associated](#managing-devices-for-testing-purposes).
+
+If the device you want to test notifications on is already added to the list, you can now test pushes:
+
+    stream chat:push:test
+
+This will do several things for you:
+
+1. Pick a random message from a channel that this user is part of
+2. Use the notification templates configured for your push providers to render the payload using this message
```

Comment:
We should mention notification templates above (or below), link to our CMS would be enough